.env.dist

```diff
@@ -1,5 +1,11 @@
 # Docker configuration
+HOST_UID=
 SERVER_HTTP_PORT=80
+POSTGRES_EXTERNAL_PORT=5432
```

Comment:
I think that for consistency we should either use _EXTERNAL_ for both postgres and nginx or not to use it in both cases.